### DISCUSSION TITLE
Documentation issues link broken

### DISCUSSION THREAD

**Discussion by ollosh** [livewire/livewire#8290] [4. General]
I want to open an issue (or PR) for documentation, but the documentation link is broken and points to [livewire 2 docs](https://github.com/livewire/docs) when opening an issue.
![image](https://github.com/livewire/livewire/assets/12979324/11aa9e2c-e6f8-4f93-af4f-199339805f45)

**Comment by joshhanley** [comment]
@ollosh Docs are now in this main repo so just open a discussion or a PR here

**Reply by ollosh** [reply]
Yeah, found it afterwards, but still think it would be nice to fix that link when opening an issue :)
